documentation/modules/con-requests-kafka-bridge.adoc

```diff
@@ -74,16 +74,50 @@ If you choose to specify a binary embedded data format, subsequent producer requ
 
 Producer requests must also provide a `Content-Type` header that corresponds to the embedded data format, for example, `Content-Type: application/vnd.kafka.binary.v2+json`.
 
+== Message format
+
+When sending messages using the `/topics/` endpoint, you enter the message payload in the `records` parameter in the request body.
+
+The `records` parameter can contain any of these optional fields:
+
+* Message `headers`
+* Message `key`
+* Message `value`
+* Destination `partition`
+
+.Example `POST` request
+[source,curl,subs=attributes+]
+----
+curl -X POST \
+  http://localhost:8080/topics/my-topic \
+  -H 'content-type: application/vnd.kafka.json.v2+json' \
+  -d '{
+    "records": [
+        {
+            "key": "my-key",
+            "value": "sales-lead-0001"
+            "partition": 2
+            "headers": [
+              {
+                "key": "key1",
+                "value": "value1"
```

Comment:
@ppatierno Was the value not supposed to be base64 encoded?

Reply:
@scholzj yes it has to be a base64 encoded value.

Reply:
I've changed this to Base64 and added a call-out.

Does the `value` on line 98 also need to be Base64 encoded? @ppatierno

Reply:
no that's the value of the message and depends on the embedded format chosen for the created consumer.